lib/template-mixins.js

```diff
@@ -528,6 +530,15 @@ module.exports = function (options, deprecated) {
             };
         };
 
+        res.locals.qs = function () {
+            return function (query) {
+                query = querystring.stringify(Object.assign({}, req.query, querystring.parse(query)));
+                var urlObj = url.parse(req.url);
+                urlObj.search = '?' + query;
+                return url.format(urlObj);
```

Comment:
I have no idea why I did that. I actually wanted the return value to be `?blah=blah`. long day